pkgs/applications/networking/browsers/surf/default.nix

```diff
@@ -1,6 +1,7 @@
-{ stdenv, fetchurl
-, pkgconfig, wrapGAppsHook
+{ lib, stdenv, fetchurl
+, pkgconfig, wrapGAppsHook, makeWrapper
 , glib, glib-networking, gsettings-desktop-schemas, gtk, libsoup, webkitgtk
+, xlibs, dmenu, findutils, gnused, coreutils
```

Comment:
Looks like `xlibs` is an alias for `xorg`. Ofborg/hydra doesn't use aliases to discourage their usage IIRC.

Reply:
Ok, fixed. Thanks!